containers/java/clipper-java-container/src/main/java/clipper/container/app/RPCTestModel.java

```diff
@@ -18,26 +19,34 @@
   }
 
   @Override
-  public FloatVector predict(DoubleVector inputVector) {
+  public SerializableString predict(DoubleVector inputVector) {
     double clipperTimestamp = inputVector.getData().get();
     RPCEvent[] eventHistory = container.getEventHistory();
-    List<Float> eventCodes = new ArrayList<>();
-    for (int i = 0; i < eventHistory.length; i++) {
+    // Begin building a JSON array
+    StringBuilder eventCodeJson = new StringBuilder("[");
```

Comment:
For future reference, don't manually construct JSON. Use a json serialization library.